src/test/kotlin/eu/europa/ec/eudi/openid4vci/examples/OfferBasedIssuanceUsingAuthorizationFlow.kt

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2023 European Commission
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.europa.ec.eudi.openid4vci.examples
+
+import eu.europa.ec.eudi.openid4vci.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.request.forms.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import org.jsoup.Jsoup
+import org.jsoup.nodes.FormElement
+import java.net.URL
+
+private val actingUser = ActingUser("tneal", "password")
+
+fun main(): Unit = runBlocking {
+    val credentialOfferUrl = "eudi-openid4ci://credentialsOffer?credential_offer=%7B%22" +
```

Comment:
Instead of `eudi-openid4ci://credentialsOffer?...`
use `eudi-openid4ci://?...`

Reply:
Done. We must update also Issuer UI to exclude it from the QR code/link generated